### PR TITLE
Added GATK module for ASEReadCounter, fixes #469

### LINKED ISSUE #469: ASEReadCounter command module
- [x] Command module
- [x] Test

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

**Comment by J35P312** [comment]
Welcome to the club jemten! =P

File: lib/MIP/Program/Variantcalling/Gatk.pm [hunk 1]
```diff
@@ -2024,4 +2024,195 @@ sub gatk_concatenate_variants {
...
+##          : $referencefile_path                    => Reference sequence file
+##          : $stderrfile_path                       => Stderrfile path
+##          : $temp_directory                        => Redirect tmp files to java temp
+
+    my ($arg_href) = @_;
+
+    ## Flatten argument(s)
+    my $downsample_to_coverage;
+    my $FILEHANDLE;
+    my $gatk_disable_auto_index_and_file_lock;
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Gatk.pm]: This one has a default. Add it under '## Defaults'. And the rest that has defaults

File: lib/MIP/Program/Variantcalling/Gatk.pm [hunk 2]
```diff
@@ -2024,4 +2024,195 @@ sub gatk_concatenate_variants {
...
+    );
+
+    ## Tool-specific options
+    push @commands, q{--sitesVCFFile} . $SPACE . $gatk_sites_vcffile;
+
+    ## Infile
+    if ($infile_path) {
+
+        push @commands, q{--variant} . $SPACE . $infile_path;
+
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Gatk.pm]: Remove line

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**